package.json

```diff
@@ -32,8 +32,8 @@
   },
   "dependencies": {
     "openzeppelin-solidity": "^2.4.0",
-    "witnet-ethereum-bridge": "^0.4.2",
-    "witnet-requests": "^0.5.0"
+    "witnet-ethereum-bridge": "~0.4.2",
```

Comment:
Upgrade to witnet-solidity-bridge ?

Reply:
> Upgrade to witnet-solidity-bridge ?

The npm package won't be renamed until a new release is made.